packages/clay-color-picker/src/Alpha.tsx

```diff
@@ -0,0 +1,111 @@
+/**
+ * SPDX-FileCopyrightText: © 2019 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {ClayInput} from '@clayui/form';
+import ClaySlider from '@clayui/slider';
+import React, {useState} from 'react';
+
+type Props = {
+	/**
+	 * Callback function for when the hue value changes
+	 */
+	onChange: (event: any) => void;
+
+	/**
+	 * The inline styles to add on ClaySlider
+	 */
+	style: {
+		color: string;
+	};
+
+	/**
+	 * The value of the alpha transparency
+	 */
+	value: number;
+};
+
+/**
+ * Renders Alpha component
+ */
+const ClayColorPickerAlpha = ({
+	style,
+	value = 0,
+	onChange = () => {},
```

Comment:
In those cases where `onChange` is required it would not be necessary to add a noop function, although we add a lot in some places in Clay at the beginning I really don't like the idea because it makes it difficult to debug bugs when you have a noop function as a default property.